src/Formio.js

```diff
@@ -670,7 +670,7 @@ export default class Formio {
       .then(() => Formio.pluginGet('request', requestArgs)
         .then((result) => {
           if (isNil(result)) {
-            return Formio.request(url, method, requestArgs.data, requestArgs.opts.header, requestArgs.opts);
+            return Formio.request(requestArgs.url, requestArgs.method, requestArgs.data, requestArgs.opts.header, requestArgs.opts);
```

Comment:
Did you mean to include this change? We can quickly merge the tree changes in but this is a major change to a foundational library and we may need to take a closer look at it.

Reply:
@randallknutson I was trying to create a separate PR for this change but accidentally added this to the semantic ui PR. Happy for you to merge the semantic one and I can create a separate PR for the other change.

Reply:
requestArg can be intercepted using the plugin architecture. Previously opts.header, data and opts where being passed into the Formio.request. By passing the url it makes it possible to modify the url if needed (we have a use case for this).

Reply:
I'd appreciate separating these two items so we can evaluate them separately. The tree for semantic is straightforward and we'll merge quickly but this change is going to require some discussion and testing to make sure we don't break anything.